application/electron/src/controllers/electron/menu.functions/handler.item.about.ts

```diff
@@ -1,11 +1,31 @@
 import { IPCMessages } from '../../../services/service.electron';
-
+import { exec, ExecException } from 'child_process';
 import ServiceElectron from '../../../services/service.electron';
 import ServicePackage from '../../../services/service.package';
-
+import Logger from '../../../tools/env.logger';
 import * as os from 'os';
 
+const logger: Logger = new Logger('HandlerItemAbout');
+
 export default function about() {
+    const platform: string = os.platform();
+    if (platform === 'darwin' || platform === 'linux') {
+        return exec('uname -v', (error: ExecException | null, stdout: string, stderr: string) => {
+            if (error) {
+                logger.error(error.message);
+                return;
+            }
+            if (stderr.trim() === '') {
+                logger.error(stderr);
+            }
```

Comment:
Why we do not "exit" on error? We still can trust output on called command even it has something in `stderr`?